bentoml/_internal/bento/bento.py

```diff
@@ -334,6 +340,21 @@ def save(
             fs.mirror.mirror(self._fs, out_fs, copy_if_newer=False)
             self._fs.close()
             self.__fs = out_fs
+            event_properties = {
+                "bento_tag": str(self.info.tag),
+                "bento_creation_timestamp": self.info.creation_time.isoformat(),
+                "bento_size": human_readable_size(calc_dir_size(bento_path)),
```

Comment:
no need for `human_readable_size` here, it will make ETL and analytic harder

Reply:
I'd recommend making it `bento_size_in_bytes` with an integer type

Reply:
done